src/components/animated-menu-button/animated-menu-button.stories.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { select, text } from '@storybook/addon-knobs';
+import notes from './notes.md';
+import AnimatedMenuButton from './animated-menu-button';
+import OptionsHelper from '../../utils/helpers/options-helper';
+import { Row } from '../row/row';
+import Link from '../link/link';
+
+storiesOf('Animated Menu Button', module)
+  .addParameters({
+    info: {
+      inline: true,
+      header: false,
+      source: false,
+      propTablesExclude: [Row, Link],
+      propTables: [AnimatedMenuButton]
+    }
+  })
+  .add('default', () => {
+    const direction = select('direction', OptionsHelper.alignBinary, OptionsHelper.alignBinary[1]);
+    const label = text('label', '');
+    const size = select('size', OptionsHelper.sizesFull, OptionsHelper.sizesFull[3]);
+
+    return (
+      <AnimatedMenuButton
```

Comment:
The animated button covers the prop types table when rendered right, and disappears left when rendered left. 

![image](https://user-images.githubusercontent.com/26925206/52701437-24b30500-2f72-11e9-8c25-4277d404e89a.png)

![image](https://user-images.githubusercontent.com/26925206/52701539-5af08480-2f72-11e9-94a5-41655d0d2ec2.png)

Reply:
I fixed it, thanks